### PR TITLE
fix(customize) only a template without plugins failed

### LINKED ISSUE #582: Can't use an image built with the only change that is a custom nginx template
I don't need any additional plugins, but only a `custom_nginx.conf`, so I did the build like this:
```
docker build --build-arg KONG_BASE=kong:2.8 --build-arg TEMPLATE=nginx.template -t kong:custom .
```
The image was built successfully, however, a container fails to start with an error:
```
nginx: [warn] the "user" directive makes sense only if the master process runs with super-user privileges, ignored in /usr/local/kong/nginx.conf:6
2022/07/25 11:24:05 [error] 1112#0: init_by_lua error: /usr/local/share/lua/5.1/kong/init.lua:543: error loading plugin schemas: on plugin '':  plugin is enabled but not installed;
no plugin found
stack traceback:
        [C]: in function 'assert'
        /usr/local/share/lua/5.1/kong/init.lua:543: in function 'init'
        init_by_lua:3: in main chunk
nginx: [error] init_by_lua error: /usr/local/share/lua/5.1/kong/init.lua:543: error loading plugin schemas: on plugin '':  plugin is enabled but not installed;
no plugin found
stack traceback:
        [C]: in function 'assert'
        /usr/local/share/lua/5.1/kong/init.lua:543: in function 'init'
        init_by_lua:3: in main chunk
Error: /usr/local/share/lua/5.1/kong/cmd/start.lua:64: failed to start nginx (exit code: 1)

  Run with --v (verbose) or --vv (debug) for more details
```
If a plugin is added during the build, just any, e.g. `--build-arg PLUGINS=kong-upstream-jwt`, then the resulting image works fine.

### PR DESCRIPTION
**Pull request by Tieske**
fixes #582

### CODE REVIEW

**Review by flrgh — DISMISSED** [review]

**Comment by Tieske** [comment]
@flrgh had to rebase and force-push after merging the other PR, now the approval is gone, and the PR needs a new approval... a 👍 would be most welcome (again)

**Review by flrgh — APPROVED** [review]

**Pull request merged**
